documentation/docs/decentralized_identifiers/resolve.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Resolve an IOTA Identity
+sidebar_label: Resolve
+description: Explain how resolving works including arguments.
+image: /img/Identity_icon.png
+keywords:
+- Resolve
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+import resolution_js from  '!!raw-loader!./../../../bindings/wasm/examples/src/resolution.js';
+import resolution_rs from  '!!raw-loader!./../../../examples/low-level-api/resolution.rs';
+
+TODO: Explain how resolving works including arguments.
```

Comment:
Not sure if we're coming back to this later or its a leftover; just a heads up!

Reply:
Thanks! But we should be coming back to this later.